### PR TITLE
Added embedColorEnd to the examples

### PR DESCRIPTION
**Pull request by Jelcoo**
Just what the title sais

### CODE REVIEW

**Comment by Jelcoo** [comment]
Made some improvements. Made the active giveaway embed green and the ended red by default

**Comment by Nico105** [comment]
image
You should change the property definition defaults too

**Comment by Jelcoo** [comment]
> image
> You should change the property definition defaults too

Good idea

**Review by Androz2091 — COMMENTED** [review]
Thank you for your contribution 👍 
I agree it's a good idea to add the embedColorEnd in the examples, but I'd like to keep red for active giveaways and black for the ended ones 😅

File: src/Constants.js [hunk 1]
```diff
@@ -98,7 +98,8 @@ exports.defaultManagerOptions = {
         botsCanWin: false,
         exemptPermissions: [],
         exemptMembers: () => false,
-        embedColor: '#FF0000',
+        embedColor: '#00FF00',
+        embedColorEnd: '#FF0000',
```
Comment (Nico105) [inline src/Constants.js:102]:
```suggestion
        embedColor: '#FF0000',
        embedColorEnd: '#000000',
```

File: src/Constants.js [hunk 2]
```diff
@@ -79,8 +79,8 @@ exports.defaultGiveawayMessages = {
  * @property {Boolean} [default.botsCanWin=false] Whether the bots are able to win a giveaway.
  * @property {Discord.PermissionResolvable[]} [default.exemptPermissions=[]] Members with any of these permissions won't be able to win a giveaway.
  * @property {Function} [default.exemptMembers] Function to filter members. If true is returned, the member won't be able to win the giveaway.
- * @property {Discord.ColorResolvable} [default.embedColor='#FF0000'] The giveaway embeds color when they are running
- * @property {Discord.ColorResolvable} [default.embedColorEnd='#000000'] The giveaway embeds color when they are ended
+ * @property {Discord.ColorResolvable} [default.embedColor='#00FF00'] The giveaway embeds color when they are running
+ * @property {Discord.ColorResolvable} [default.embedColorEnd='#FF0000'] The giveaway embeds color when they are ended
```
Comment (Nico105) [inline src/Constants.js:83]:
```suggestion
 * @property {Discord.ColorResolvable} [default.embedColor='#FF0000'] The giveaway embeds color when they are running
 * @property {Discord.ColorResolvable} [default.embedColorEnd='#000000'] The giveaway embeds color when they are ended
```

File: examples/simple.js [hunk 3]
```diff
@@ -14,7 +14,8 @@ const manager = new GiveawaysManager(client, {
     default: {
         botsCanWin: false,
         exemptPermissions: ['MANAGE_MESSAGES', 'ADMINISTRATOR'],
-        embedColor: '#FF0000',
+        embedColor: '#00FF00',
+        embedColorEnd: '#FF0000',
```
Comment (Nico105) [inline examples/simple.js:18]:
```suggestion
        embedColor: '#FF0000',
        embedColorEnd: '#000000',
```

File: examples/sharding.js [hunk 4]
```diff
@@ -22,7 +22,8 @@ const manager = new GiveawayManagerWithShardSupport(client, {
     default: {
         botsCanWin: false,
         exemptPermissions: ['MANAGE_MESSAGES', 'ADMINISTRATOR'],
-        embedColor: '#FF0000',
+        embedColor: '#00FF00',
+        embedColorEnd: '#FF0000',
```
Comment (Nico105) [inline examples/sharding.js:26]:
```suggestion
        embedColor: '#FF0000',
        embedColorEnd: '#000000',
```

File: examples/custom-databases/mysql.js [hunk 5]
```diff
@@ -99,7 +99,8 @@ const manager = new GiveawayManagerWithOwnDatabase(client, {
     default: {
         botsCanWin: false,
         exemptPermissions: [ 'MANAGE_MESSAGES', 'ADMINISTRATOR' ],
-        embedColor: '#FF0000',
+        embedColor: '#00FF00',
+        embedColorEnd: '#FF0000',
```
Comment (Nico105) [inline examples/custom-databases/mysql.js:103]:
```suggestion
        embedColor: '#FF0000',
        embedColorEnd: '#000000',
```

File: examples/custom-databases/mongodb.js [hunk 6]
```diff
@@ -60,7 +60,8 @@ const manager = new GiveawayManagerWithOwnDatabase(client, {
     default: {
         botsCanWin: false,
         exemptPermissions: [ 'MANAGE_MESSAGES', 'ADMINISTRATOR' ],
-        embedColor: '#FF0000',
+        embedColor: '#00FF00',
+        embedColorEnd: '#FF0000',
```
Comment (Nico105) [inline examples/custom-databases/mongodb.js:64]:
```suggestion
        embedColor: '#FF0000',
        embedColorEnd: '#000000',
```

File: README.md [hunk 7]
```diff
@@ -48,7 +48,8 @@ const manager = new GiveawaysManager(client, {
     default: {
         botsCanWin: false,
         exemptPermissions: ['MANAGE_MESSAGES', 'ADMINISTRATOR'],
-        embedColor: '#FF0000',
+        embedColor: '#00FF00',
+        embedColorEnd: '#FF0000',
```
Comment (Nico105) [inline README.md:52]:
```suggestion
        embedColor: '#FF0000',
        embedColorEnd: '#000000',
```

File: README.md [hunk 8]
```diff
@@ -352,7 +353,8 @@ const manager = new GiveawayManagerWithOwnDatabase(client, {
...
```
Comment (Nico105) [inline README.md:357]:
```suggestion
        embedColor: '#FF0000',
        embedColorEnd: '#000000',
```

File: README.md [hunk 9]
```diff
@@ -395,7 +397,8 @@ const manager = new GiveawayManagerWithShardSupport(client, {
...
```
Comment (Nico105) [inline README.md:401]:
```suggestion
        embedColor: '#FF0000',
        embedColorEnd: '#000000',
```

**Review by Nico105 — DISMISSED** [review]
According to Androz's color preference

**Comment by Nico105** [comment]
Thank you 👍

**Pull request merged**